corehq/apps/users/views/mobile/users.py

```diff
@@ -924,6 +926,32 @@ def _modify_user_status(request, domain, user_id, is_active):
     })
 
 
+@require_can_edit_commcare_users
+@require_POST
+@location_safe
+def activate_connectid_link(request, domain, user_id):
+    return _toggle_connectid_link(request, domain, user_id, True)
+
+
+@require_can_edit_commcare_users
+@require_POST
+@location_safe
+def deactivate_connectid_link(request, domain, user_id):
+    return _toggle_connectid_link(request, domain, user_id, False)
+
+
+def _toggle_connectid_link(request, domain, user_id, is_active):
+    if not toggles.COMMCARE_CONNECT.enabled(domain):
+        return HttpResponseBadRequest()
+    user = CommCareUser.get_by_user_id(user_id, domain)
```

Comment:
The tradeoff is that we lookup and send django user IDs for all users in the `paginate_mobile_workers` so that the caller knows the django userid to pass it to this view.